src/_core/polling.ts

```diff
@@ -1,6 +1,6 @@
 export function startPolling(
   ms: number,
-  fn: (abortSignal: AbortSignal) => Promise<void>
+  fn: (abortSignal: AbortSignal) => Promise<unknown>
```

Comment:
Is this change really needed? We don't expect function to return anything, because it's a side-effect function, so I think it's better to stick to `void` here